todomanager.py

```diff
@@ -0,0 +1,25 @@
+import uuid
+
+class Task:
+
+    def __init__(self, content):
+        self.content = content
+
+class TodoManager:
+
+    def __init__(self):
+        self.tasks = {}
+
+    def addTask(self, task):
+        if not isinstance(task, Task):
+            raise ValueError(
+                    "Expected instance of Task, not {}, of type {}"
```

Comment:
This is a bit confusing.  Is there any reason to print the object itself?  Perhaps just `"Expected provided task to be an instance of Task, not of type {}".format(type(task))` would be fine